### PR TITLE
(issue 889): replaced pytest-pep8 by pytest-flake8

### PR DESCRIPTION
**Pull request by alixdamman**

### CODE REVIEW

File: larray/tests/test_session.py [hunk 1]
```diff
@@ -7,7 +7,8 @@
 import pandas as pd
 import pytest
 
-from larray.tests.common import (assert_array_nan_equal, inputpath, tmp_path, meta,
+from larray.tests.common import meta            # noqa: F401
```
Comment (gdementen) [inline larray/tests/test_session.py:10]: why do we need to import meta if it is unused?

**Review by gdementen — COMMENTED** [review]
LGTM

File: larray/tests/test_session.py (hunk 1 above)
Comment (alixdamman) [reply larray/tests/test_session.py:10]: `meta` is a [pytest fixture](https://docs.pytest.org/en/stable/fixture.html#fixtures-as-function-arguments) (`meta` is defined in the /tests/common.py module) and is actually used as argument in many unittests. 
It's just that flake8 is confused with pytest fixtures.

File: larray/tests/test_excel.py [hunk 2]
```diff
@@ -299,12 +295,12 @@ def test_excel_report_setting_template():
 def test_excel_report_sheets():
     report = ExcelReport()
     # test adding new sheets
-    sheet_population = report.new_sheet('Population')
-    sheet_births = report.new_sheet('Births')
-    sheet_deaths = report.new_sheet('Deaths')
+    sheet_population = report.new_sheet('Population')       # noqa: F841
```
[the remark this answers is not in the document]
Comment (alixdamman) [reply larray/tests/test_excel.py:298]: Good point. We actually don't need to store the result in a variable.

**Comment by alixdamman** [comment]
Thanks for the review

**Pull request merged**